README.md

```diff
@@ -53,40 +34,23 @@ git branch -u upstream/main main
   <span> · </span>
   <a href="https://qiskit.org/advocates/">Advocates</a>
   <span> · </span>
-  Support: <a href="http://qisk.it/join-slack">Slack</a>
+  <a href="https://qiskit.org/ecosystem/">Ecosystem</a>  
```

Comment:
I think that we should display all of the top navbar links here, and ideally in the same order as in the website. These are missing:

- Events
- Providers

Reply:
Added in [this commit](https://github.com/Qiskit/qiskit.org/pull/3172/commits/e663ede4b578cbd2c4034f7a29d7e5b5588449d1)

![Screenshot 2023-05-08 at 6 27 24 AM](https://user-images.githubusercontent.com/6276074/236836737-716efe61-8201-490c-a20e-e602692fb637.png)